core/replicated_map.go

```diff
@@ -39,7 +39,7 @@ type ReplicatedMap interface {
 	// PutWithTtl associates a given value to the specified key and replicates it to the
 	// cluster. If there is an old value, it will be replaced by the specified
 	// one and returned from the call. In addition, you have to specify a ttl and its time unit
-	// to define when the value is outdated and thus should be removed from the
+	// to define when the value is outdated and this should be removed from the
```

Comment:
I think "thus" was used correctly in that context, tho it is redundant. You can either change "this" to "thus", or just remove "this".